app/src/server/tasks/relabelDatasetEntryTask.ts

```diff
@@ -106,6 +108,8 @@ export const relabelDatasetEntry = defineTask<RelabelDatasetEntryJob>({
       ]);
 
       if (newDatasetEntry.split === "TEST") {
+        await copyDatasetEvalDatasetEntries(datasetEntry.id, newDatasetEntry.id);
+        await updatePruningRuleMatches(datasetEntry.datasetId, new Date(0), [newDatasetEntry.id]);
```

Comment:
Don't we want to update pruning rule matches even if the split is TRAIN?

Reply:
Yes we do, good catch.